scripts/secrets/azure.msf.env

```diff
@@ -211,6 +211,6 @@ MSF_DHIS2_PASSWORD=Admin123
 # Traefik Configuration
 # ==============================================================================
 
-TRAEFIK_API_URL=api.traefik.me
+TRAEFIK_HOSTNAME=api-172-17-0-1.traefik.me
```

Comment:
Isn't this ip usually auto-generated? Does adding this force it to use this exact number?

Reply:
Since the Ip is dynamic depening on the OS, we provide an initial value since it has to resolve when running in docker compose files